src/site/views/event/tmpl/default.php

```diff
@@ -178,7 +178,10 @@
 										{
 											$person['Dietary'] = "";
 										}
-
+										if ($person['Details'] == "{\"addons\":[]}")
```

Comment:
Improvement would be to parse it to JSON and then check that the "addons" list is empty.

Reply:
possibly yeah, but time taken to parse into a new variable, and check a list, vs just checking the string that comes out of the Model, is why i just did a direct compare.